packages/publish/src/MosaicPublisher.ts

```diff
@@ -101,8 +101,7 @@ export class MosaicPublisher {
     mockCanvas(globalThis.window);
 
     // Load the visualization in the DOM and gather interactors/inputs
-    // TODO: fix type issue with astToDOM to remove the any cast
-    const { element } = await astToDOM(this.ast, { api: this.ctx.api } as any);
+    const { element } = await astToDOM(this.ast, { api: this.ctx.api });
```

Comment:
I don't think this file should be in this PR, as it pertains to the WIP publish package.

Reply:
Would it be best for me to remove these changes and make them in the publish package PR (changing the base of that PR to this branch and the base of this PR to main)? As of right now these changes would be merged into the publish package PR which is why I included this.

Reply:
As I see. I was thinking of merging into `main`, hence the comment. I still think that would be ideal.

Reply:
Gotcha, will address accordingly 👍

Reply:
Should be fixed now! See #682 for above changes and they shouldn't be in this PR anymore